doc/btrfs-multiple-devices.md

```diff
@@ -0,0 +1,101 @@
+
+Multiple Devices Btrfs
+======================
+
+
+Mounting Degraded Btrfs
+-----------------------
+
+Mounting a degraded btrfs requires the 'degraded' mount option.
+
+Using the 'degraded' mount option in /etc/fstab by default was not recommended
+by btrfs experts (on IRC).
```

Comment:
can you name it here? so we can ask them in future?

Reply:
No, since I am not sure whether that is legally OK.